resources/views/projects/milestones/edit.blade.php

```diff
@@ -0,0 +1,86 @@
+@extends('layouts.app')
```

Comment:
This file looks almost exactly the same as the `create` file. Why are we not using the same one?